Applications/WebPortal/WebApp/library/source/pages/account_page/risk_page/risk_parameters_view.tsx

```diff
@@ -56,13 +56,15 @@ export class RiskParametersView extends React.Component<Properties> {
           <MoneyField
             className={css(RiskParametersView.STYLE.inputBox)}
             value={this.props.parameters.buyingPower}
+            min={RiskParametersView.ZERO_MONEY}
```

Comment:
You don't need a new variable, you should just be able to use` Nexus.Money.ZERO`.